components/ServiceSection/ServiceSectionContent/ServiceSectionContent.styles.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+
+export const ServiceSectionContainer = styled.div`
+  display: flex;
+  gap: 10.375rem;
+  flex-direction: row;
+  justify-content: center;
+  padding: 0 22.5rem 7.5rem;
+
+  background: linear-gradient(
+      180deg,
+      #eaf8f9 50.93%,
+      rgba(234, 248, 249, 0) 80.88%
+    ),
+    url(/Image/ServiceSectionBackground.png);
+  mix-blend-mode: multiply;
+`;
+
+export const ServiceSectionMessageContainer = styled.div`
+  margin-top: 7.188rem;
+  text-align: center;
+`;
+
+export const ServiceSectionHeading = styled.h1`
+  height: 4.375rem;
+  width: 27.125rem;
+  margin: auto;
+
+  font-weight: 700;
+  color: #043236;
+  font-size: 3.75rem;
+  line-height: 4.375rem;
+  text-transform: uppercase;
+  font-family: "ITC Avant Garde Gothic W01 Bd";
+`;
+
+export const ServiceSectionDescription = styled.p`
+  width: 627px;
+  height: 56px;
+  margin: auto;
+  margin-top: 30px;
+  margin-bottom: 50px;
+
+  font-family: "ITC Avant Garde Gothic Std";
```

Comment:
Use the following font family for this `font-family:"ITC Avant Garde Gothic W01 Bk";` I have a paid font theme which is not correctly reflected in Figma.

Reply:
Do you want this just for the description below the heading or on the card descriptions as well?